Tests-Exercise02.postman_collection.json

```diff
@@ -148,6 +153,64 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "/messages (With number included)",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "urlencoded",
```

Comment:
be careful, your contect type is not correct. 
You need to change to application/json